torch/__init__.py

```diff
@@ -1686,5 +1686,20 @@ def _sparse_coo_tensor_unsafe(*args, **kwargs):
     return torch.sparse_coo_tensor(*args, **kwargs)
 
 
+if sys.executable != 'torch_deploy':
+    class _TritonLibrary(object):
+        lib = torch.library.Library("triton", "DEF")
+        ops_table = {}
+
+        @classmethod
+        def probablyRegisterOp(cls, op_key, full_schema, op_impl, dispatch_key):
```

Comment:
Ho a cache? When do you need to use the returned value?

Reply:
The returned value is used in tests to check whether it is registered and whether dispatch in `nn.Linear` actually worked.

Reply:
So this API is indeed designed like this just for testing?

Reply:
Looking at the comments below, I think Nikita is confused as well about this api :p

Reply:
Yeah, `Library` does not allow to query registered `impl` and it is not possible to force re-register, so that there is an interface that allows to check whether the dispatch actually happened.